packages/block-editor/src/components/url-input/index.js

```diff
@@ -111,15 +111,18 @@ function URLInput( {
 	const suggestionNodes = useRef( [] );
 	const instanceId = useInstanceId( URLInput );
 
-	const { fetchLinkSuggestions } = useSelect(
+	// Allow passing in a __experimentalFetchLinkSuggestions function as a prop
+	// otherwise use the __experimentalFetchLinkSuggestions in the block editor settings.
+	const {
+		fetchLinkSuggestions = __experimentalFetchLinkSuggestions,
```

Comment:
@ntsekouras  I opted to go back to using the default in the return object since `useSelect` is mocked in the LinkControl test. The issue is that somewhere up the component tree the function is called `fetchSearchSuggestions` so that's mocked in the LinkControl here [#](https://github.com/jhnstn/gutenberg/blob/master/packages/block-editor/src/components/link-control/test/index.js#L28) . I tried previously to update the test but I couldn't get it to work and what ever I added seemed odd since the `URLInput` component was not being tested directly.

I'm happy to try again to get that test to pass without the default but I'm not sure it's worth the time.

Reply:
Since you added the comment is okay.